cairosvg/surface/__init__.py

```diff
@@ -323,6 +331,9 @@ def draw(self, node):
                 self.context.clip()
                 self.context.set_fill_rule(cairo.FILL_RULE_WINDING)
 
+        save_cursor = (self.cursor_position, self.cursor_d_position,
```

Comment:
It should be:
save_cursor = copy.deepcopy((self.cursor_position, self.cursor_d_position, self.text_path_width))